web/src/main/java/co/mobileaction/example/web/controller/PostController.java

```diff
@@ -34,11 +35,18 @@ public ResponseEntity<List<Post>> getPosts(@PageableDefault(size = 10)
         return ResponseEntity.ok(postService.findPosts(pageable));
     }
 
-    @DeleteMapping("{postId}")
+    @DeleteMapping("/{postId}")
     public ResponseEntity<Boolean> deletePost(@PathVariable Long postId)
     {
+        //postService.deletePost(postId);

```

Comment:
we can remove this commented out code, there is already the same line next to it.